src/utils/index.js

```diff
@@ -22,19 +31,48 @@ export const calculateMargin = ({
   mRight,
   mHorizontal,
   mVertical,
-  margin
+  margin,
+}: {
+  mTop: number,
+  mBottom: number,
+  mLeft: number,
+  mRight: number,
+  mHorizontal: number,
+  mVertical: number,
+  margin: number,
 }) => ({
   marginTop: margin || mVertical || mTop || 0,
   marginBottom: margin || mVertical || mBottom || 0,
   marginLeft: margin || mHorizontal || mLeft || 0,
-  marginRight: margin || mHorizontal || mRight || 0
+  marginRight: margin || mHorizontal || mRight || 0,
 });
 
-export const generateLotteryNumbers = (min, max, quantity) => {
-  const numbers = Array(quantity).fill(0);
-  for (let index = 0; index < numbers.length; index += 1) {
-    numbers[index] = Math.floor(Math.random() * (max - min + 1) + min);
+export const generateLotteryNumbers = (
+  min: number,
+  max: number,
+  quantity: number,
+) => {
+  const numbers = [];
+  while (numbers.length < quantity) {
+    const number = Math.floor(Math.random() * (max - min + 1) + min);
+    if (numbers.indexOf(number) === -1) {
+      numbers.push(number);
+    }
   }
   numbers.sort((a, b) => a - b);
   return numbers;
 };
+
+export const hexToRGB = (hex: string, alpha: number) => {
+  const r = parseInt(hex.slice(1, 3), 16);
```

Comment:
[0-9A-Fa-f]{6}
```suggestion
  const r = parseInt(hex.slice(1, 3), 16);
```
![image](https://user-images.githubusercontent.com/18670067/64083424-4e593100-ccf6-11e9-8dbf-aa460c3a31c4.png)


![image](https://user-images.githubusercontent.com/18670067/64083414-2e297200-ccf6-11e9-934c-aa2e522201ec.png)